source/_patterns/03-organisms/sections/block-group-2up.twig

```diff
@@ -0,0 +1,17 @@
+<section class="c-block-group c-block-group--2up o-section {{ block_group_classes }}">
+
+  <div class="c-block-group__inner">
```

Comment:
what purpose is this `__inner` element serving? `.c-block-group` is applying the same flex container rules as `__inner`, and `__inner` is the only child element.

could you achieve the same results by removing the `__inner` element and moving `__inner`'s media queries to `--2up`?

Reply:
I think I was thinking it might have a heading outside the flexed area at some point. I'll worry about that then. Consider it gone!